sublime-package.json

```diff
@@ -0,0 +1,77 @@
+{
+    "contributions": {
+        "settings": [
+            {
+                "file_patterns": [
+                    "/LSP-elm.sublime-settings"
+                ],
+                "schema": {
+                    "$id": "sublime://settings/LSP-elm",
+                    "definitions": {
+                        "PluginConfig": {
+                            "properties": {
+                                "initializationOptions": {
+                                    "type": "object",
+                                    "additionalProperties": false,
+                                    "properties": {
+                                        "trace.server": {
+                                            "type": "boolean",
+                                            "markdownDescription": "Enable/disable trace logging of client and server communication"
```

Comment:
Not sure this setting is actually supported but could add a trailing dot in the description for consistency.

Reply:
ab5545c

Reply:
> Not sure this setting is actually supported

Good question. I don't see any change in the logs panel or in the ST console when I turn this setting on or off.